src/hooks/fetchUtils.test.ts

```diff
@@ -0,0 +1,229 @@
+import { NODE_MAJOR_VERSION } from '../../testUtils/nodeVersion';
```

Comment:
Why did you move these tests to the `fetchUtils.test.ts` file? We usually keep the test file name the same as the file with the source code it tests (I would expect a `fetchUtils.ts` file to exist).

Please move these tests back to the `fetch.test.ts` file

Reply:
Now there is a huge diff, I can't see what is changed in the test and what isn't

Reply:
Discussed that. Adding extra comments to the code.